DetectorDescription/DDCMS/plugins/dd4hep/DDDefinitions2Objects.cc

```diff
@@ -2058,7 +2058,7 @@ static long load_dddefinition(Detector& det, xml_h element) {
 
     string fname = xml::DocumentHandler::system_path(element);
     bool open_geometry = dddef.hasChild(DD_CMU(open_geometry)) ? dddef.child(DD_CMU(open_geometry)) : true;
-    bool close_geometry = dddef.hasChild(DD_CMU(close_geometry)) ? dddef.hasChild(DD_CMU(close_geometry)) : false;
+    bool close_geometry = dddef.hasChild(DD_CMU(close_geometry)) ? dddef.hasChild(DD_CMU(close_geometry)) : true;
```

Comment:
@ianna why not 
`bool close_geometry = true` ?
This must be equivalent.
Or did you mean:
 `bool close_geometry = dddef.hasChild(DD_CMU(close_geometry)) ? dddef.**child**(DD_CMU(close_geometry)) : true;`
instead?

Reply:
Closing (or not) geometry should be controlled via configuration, so it should be: 
```c++
bool close_geometry = dddef.hasChild(DD_CMU(close_geometry)) ? dddef.hasChild(DD_CMU(close_geometry)) : false;
```
but there are places in the configurations where the `close_geometry` tag is not present, but needed. For example, in the MagneticField geometry. It needs a special "hidden" volume.